ratis-server/src/main/java/org/apache/ratis/server/impl/LogAppender.java

```diff
@@ -221,11 +221,12 @@ private TermIndex getPrevious(long nextIndex) {
     return null;
   }
 
-  protected AppendEntriesRequestProto createRequest(long callId) throws RaftLogIOException {
+  protected AppendEntriesRequestProto createRequest(long callId,
+      boolean heartbeat) throws RaftLogIOException {
```

Comment:
Can we rename `heartbeat ` to `excludeLogEntries `?

Reply:
addressed it in the latest commit.